doc/translation-status.rst

```diff
@@ -3,31 +3,31 @@
 +---------------------------------------------+---------------------------------------------+
 | Site                                        | Translations                                |
 +=============================================+=============================================+
-| index.rst                                   | zh_CN, de_DE                                |
```

Comment:
This probably highlights a bug in that code, the output is not stable, changes should not be there. I saw you do translation stuff in here, would be better to do that on translation branch separately, but I will take it.